.gitignore

```diff
@@ -11,4 +11,5 @@ __pycache__
 app/public/appyters.json
 data/
 docker-compose.yml
-venv
\ No newline at end of file
+venv
+appyters/Tumor_Gene_Target_Screener/Mammalia/Homo_sapiens.gene_info.tsv
```

Comment:
Please move this to `appyters/Tumor_Gene_Target_Screener/.gitignore` and just ignore `Mammalia`